runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsBrotherSlainOverlay.java

```diff
@@ -72,9 +72,11 @@ public Dimension render(Graphics2D graphics)
 			potential.setHidden(true);
 		}
 
+		float BrotherReward = 0;
```

Comment:
We can avoid adding a new variable by moving `rewardPercent`'s definition to this line and simply adding `0.2f` to the reward percent for each brother that's killed.

Reply:
> We can avoid adding a new variable by moving `rewardPercent`'s definition to this line and simply adding `0.2f` to the reward percent for each brother that's killed.

I don't see how one would do this without doing some bad variable use and have 'rewardPercent' represent different things at different points. The brother variable (alpha) is in the denominator so there is no clean way to separate it. If you see something i don't, could you please elaborate?